cfme/tests/webui/test_general_ui.py

```diff
@@ -366,6 +370,75 @@ def test_vm_right_size_recommendation_back_button(setup_provider, full_template_
     assert view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1627387])
+@pytest.mark.customer_scenario
+@pytest.mark.provider([BaseProvider], selector=ONE)
+def test_misclicking_checkbox_vms(appliance, setup_provider, provider):
+    """
+    Bugzilla:
+        1627387
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/8h
+        setup:
+            1. Add a provider.
+        testSteps:
+            1. Navigate to All VMs/Instances page.
+            2. Select the list view.
+            3. Click on the first column, it contains checkbox.
+            4. Assert that nothing happens and the page stays the same.
+    """
+    collection = appliance.provider_based_collection(provider)
+    view = navigate_to(collection, "All")
+    if view.toolbar.view_selector.selected != "List View":
+        view.toolbar.view_selector.select("List View")
+
+    table = Table(view, '//*[@id="miq-gtl-view"]//table')
```

Comment:
@valaparthvi No need of this as per our implementation for list view you can access table like
```
view.entities.elements
```